policyengine_us/variables/gov/states/id/tax/income/credits/id_aged_or_disabled_credit.py

```diff
@@ -0,0 +1,31 @@
+from policyengine_us.model_api import *
+
+
+class id_aged_or_disabled_credit(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Idaho aged or disabled credit"
+    unit = USD
+    definition_period = YEAR
+    defined_for = StateCode.ID
+
+    def formula(tax_unit, period, parameters):
+        person = tax_unit.members
+        age = person("age", period)
+        head = person("is_tax_unit_head", period)
+        spouse = person("is_tax_unit_spouse", period)
+        disabled = person("is_disabled", period)
+        p = parameters(
+            period
+        ).gov.states.id.tax.income.credits.aged_or_disabled
+        age_eligible = (age >= p.age_eligibility) & ~(head | spouse)
+        eligible = age_eligible | disabled
+        total_eligible = sum(eligible)
```

Comment:
@MaxGhenis for some reason `tax_unit.sum` does not seem to actually sum up the array here and only the `sum` function does. Is there a reason that we missed? (All tests pass)